### ISSUE TITLE
Add videos, interviews, and articles to Core and Elective sections of curriculum.

### ISSUE THREAD

**Issue by unshakenme** [unshakenme/OSPAT#3]
The curriculum is incomplete. We need recommendations for additional content, especially articles, videos/lectures, and interviews that will help enhance student learning.

**Leave a comment directly on this issue with ideas!**

**Comment by CheekeyMonkey** [comment]
Happy to help. I think it would be best to not just add links of things to read but more write a summary and then add a link to each point. For instance... 

Whilst the pharmaceutical industry markets that psychedelics can cure everything from obesity to hair loss, in reality, there is little evidence that they can treat any other conditions apart from mental health issues. In regards to mental health, psychedelics as a group have a very similar effect[[1]][[2]][[3]]

[[1]]Direct comparison of the acute effects of lysergic acid diethylamide and psilocybin in a double-blind placebo-controlled study in healthy subjects. Holze, F., Ley, L., Müller, F. et al. Neuropsychopharmacol. 47, 1180–1187 (2022). https://doi.org/10.1038/s41386-022-01297-2
 
[[2]]Double-blind comparison of the two hallucinogens psilocybin and dextromethorphan: similarities and differences in subjective experiences. Carbonaro TM, Johnson MW, Hurwitz E, Griffiths RR. Psychopharmacology (Berl). 2018 Feb;235(2):521-534. doi: 10.1007/s00213-017-4769-4. Epub 2017 Nov 7. PMID: 29116367; PMCID: PMC6645364.

[[3]]  Comparison of the Reactions Induced by Psilocybin and LSD-25 in Man. From the National Institute of Mental Health, Addiction Research Center, U. S. Public Health Service, Lexington, Kentucky by HARRIS ISBELL 1959

This would make for better readability.

**Comment by unshakenme** [comment]
Absolutely, that would be wonderful. If you have time to do that, please do and submit the changes as a PR. I'm glad to have you here!

**Comment by CheekeyMonkey** [comment]
This will take a lot of work and review. I'm helping on this open source project https://burnzero.com/Psychedelics aswell to get a basis of the text with associated references, perhaps we work on that site via their redidt page: https://www.reddit.com/r/burnzero/. Once set up we transfer to here as a pull request.

**Comment by unshakenme** [comment]
Yes this will make up the core of the project and definitely needs to be done in a thoughtful way. 

I took a look at that project and while there may be some useful content for our purposes, it’s definitely not completely aligned. If we were to do that would we be able a pick and choose what we copy over?

**Comment by CheekeyMonkey** [comment]
Yes, its fine to use it as long as we attribute.
